docs/user/dashboard/dashboard-controls.asciidoc

```diff
@@ -0,0 +1,136 @@
+** For option list controls on _string_ and _IP address_ type fields, you can define how the control's embedded search should behave: 
+
+*** **Prefix**: Show options that _start with_ the entered value.
+*** **Contains**: Show options that _contain_ the entered value. This setting option is only available for _string_ type fields. Results can take longer to show with this option.
+*** **Exact**: Show options that are a 100% match with the entered value.
++
+TIP: The search is not case sensitive. For example, searching for `ios` would still retrieve `iOS` if that value exists.
+
+** *Ignore timeout for results* delays the display of the list of values to when it is fully loaded. This option is useful for large data sets, to avoid missing some available options in case they take longer to load and appear when using the control.
+
+* For Range sliders, set the step size. The step size determines the number of steps of the slider. The smaller the step size is, the more steps there is in the slider.
+
+. Click *Save and close*. The control can now be used.
+
+. Consider the position of the control if you have several controls active on the dashboard. Controls are applied from left to right, which can change the options available depending on their position when the <<configure-controls-settings,Chain controls>> setting is enabled.
+
+. Save the dashboard.
+
+[float]
+[[add-time-slider-controls]]
+==== Add time slider controls
+
+You can add one interactive time slider control to a dashboard.
+
+. Open or create a new dashboard. 
+
+. In the dashboard toolbar, click *Controls* > *Add time slider control*.
+
+. The time slider control uses the time range from the global time filter. To change the time range in the time slider control, <<set-time-filter,change the global time filter>>. 
+
+. Save the dashboard. The control can now be used.
+
+
+
+[float]
+[[configure-controls-settings]]
+==== Configure the controls settings
+
+Several settings that apply to all controls of a same dashboard are available.
+
+. In the dashboard toolbar, click *Controls*, then select *Settings*.
+
+. On the *Control settings* flyout, configure the settings:
+
+* *Label position* &mdash; Specify where the control label appears.
+
+* *Filtering* settings:
+
+** **Apply global filters to controls** &mdash; Define whether controls should ignore or apply any filter specified in the main filter bar of the dashboard.
+** **Apply global time range to controls** &mdash; Define whether controls should ignore or apply the main time range specified for the dashboard. Note that <<add-time-slider-controls,time slider controls>> rely on the global time range and don't work properly when this option is disabled.
+
+* *Selection* settings:
+
+** *Validate user selections* &mdash; When selected, any selected option that results in no data is ignored.
+** *Chain controls* &mdash; When selected, any selected options in one control narrows the available options in the next control.
```

Comment:
How about "When selected, any selected options in one control narrows the available options in controls to the right of the control with the selected options."

Reply:
I edited it this way, how does that sound to you?:
"When selected, controls are applied sequentially from left to right, and line by line. Any selected options in one control narrows the available options in the next control."

Reply:
I like that a lot. Thanks for adding